hs/app/reach/Main.hs

```diff
@@ -896,6 +896,7 @@ compile = command "compile" $ info f d where
             -e REACH_CONNECTOR_MODE \
             -e "REACHC_ID=$${ID}" \
             -e "CI=$ci'" \
+            -p 3001:3001 \
```

Comment:
This is very dangerous. For now, make it so this only happens when you do `--sim` and make a note to do it better in the future